rotkehlchen/data_migrations/migrations/migration_5.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def data_migration_5(write_cursor: 'DBCursor', rotki: 'Rotkehlchen') -> None:  # noqa: E501 # pylint: disable=unused-argument
+    """
+    Update view for amm_swaps making sure that the link is a string instead of blob
+    """
+    log.debug('Starting migration 5')
+    write_cursor.execute('DROP VIEW combined_trades_view;')
+    log.debug('Creating combined_trades_view for migration 5')
```

Comment:
Do you need those debug statements? IF you think they are needed then you need one at start and one at the end. Why put one after the drop view? It looks like this was for your own debugging purposes.

```suggestion
    write_cursor.execute('DROP VIEW combined_trades_view;')
```